src/main/java/competition/subsystems/carousel/commands/TurnRightCarouselCommand.java

```diff
@@ -24,6 +27,10 @@ public void initialize(){
     }
 
     public void execute(){
```

Comment:
We should revert the changes to this file. The carousel works a little differently than you imagine.